### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by thesteve0**
Trying to get the appdev image to build  - needs someone to run the test (I am only on windows machines today)

### CODE REVIEW

File: centos7/11/Dockerfile.postgres-appdev.centos7 [hunk 1]
```diff
@@ -27,9 +27,10 @@ RUN yum -y update && yum -y install epel-release \
     hostname \
     procps-ng  \
     rsync openssl\
- && yum -y install postgresql11-server postgresql11-contrib postgresql11 \
+ && yum -y um -y install --disablerepo="${PGDG_REPO_DISABLE}" \
```
Comment (jkatz) [inline centos7/11/Dockerfile.postgres-appdev.centos7:30]: This line breaks. It should be:

```docker
&& yum -y install --disablerepo="${PGDG_REPO_DISABLE}" \
```

**Review by jkatz — CHANGES_REQUESTED** [review]
Tried building on CentOS 7. A few comments:

1. With the correction in the file mentioned in the comments, I get up to this error before it fails:

```sh
Transaction check error:
  file /usr/pgsql-11/lib/address_standardizer.so conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/bitcode/address_standardizer.index.bc conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/bitcode/address_standardizer/analyze.bc conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/bitcode/address_standardizer/parseaddress-api.bc conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/bitcode/address_standardizer/std_pg_hash.bc conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/liblwgeom.so conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/postgis-2.4.so conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/postgis_topology-2.4.so conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/lib/rtpostgis-2.4.so conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/address_standardizer.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/address_standardizer_data_us.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/address_standardizer_data_us.sql conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/postgis.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/postgis_sfcgal.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/postgis_tiger_geocoder.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64
  file /usr/pgsql-11/share/extension/postgis_topology.control conflicts between attempted installs of postgis25_11-2.5.3-5.rhel7.x86_64 and postgis24_11-2.4.8-5.rhel7.x86_64

Error Summary
-------------

error building at STEP "RUN yum -y update && yum -y install epel-release  && yum -y update glibc-common  && yum -y install bind-utils     gettext     hostname     procps-ng      rsync openssl && yum -y install --disablerepo="${PGDG_REPO_DISABLE}"     postgresql11-server postgresql11-contrib postgresql11 postgresql11-plpython    R-core libRmath plr11     postgis24_11 postgis24_11-client  pgrouting_11 && yum -y clean all": error while running runtime: exit status 1
make: *** [postgres-appdev] Error 1
```

It looks like both PostGIS 2.5 and PostGIS 2.4 are in there 🤔 

2. I would also suggest updating the corresponding UBI7 image in `ubi7/11/Dockerfile.postgres-appdev.ubi7`

Thanks!

**Comment by thesteve0** [comment]
I don't understand how the postgis container builds properly then

**Comment by thesteve0** [comment]
Nevermind , just read to the end I will try fixing the typo. Not moving to
UBI until the core team does

**Comment by jkatz** [comment]
@thesteve0 RE UBI7, there is already a container in there for it:

https://github.com/CrunchyData/crunchy-containers/blob/develop/ubi7/11/Dockerfile.postgres-appdev.ubi7

If you do not want to use it, I'd suggest we remove it.

**Comment by jkatz** [comment]
Closed in favor of #1146 and #1147

**Pull request closed without merging**